Tasks/Common/webdeployment-common/WebConfigTemplates/node

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
i had got a comment to include templates in common folder so that it can be used in future by other tasks such as  IIS App Deploy task.